app/components/Snackbar/index.js

```diff
@@ -30,17 +30,23 @@ const snackbarClasses = ({ type }) => ({
 
 @autobind
 class Snackbar extends React.Component {
-
   constructor(props) {
     super(props);
     this.hideTimer = null;
     this.state = {
       message: props.messages.length > 0
         ? props.messages[props.messages.length-1]
-        : null
+        : null,
+      delay: 250, // delay in ms to render component
+      isWaiting: true,
     };
   }
 
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({isWaiting: false}); }, this.state.delay);
```

Comment:
This doesn't work as you intend it to work.

The snackbar component only gets mounted once (since it's added to the App container). You can easily verify this by adding a console message inside the setTimeout() callback: isWaiting only gets set to false once.